moto/elasticache/models.py

```diff
@@ -137,13 +141,91 @@ def get_tags(self) -> List[Dict[str, str]]:
         return self.tags
 
 
+class CacheSubnetGroup(BaseModel):
+    def __init__(
+        self,
+        account_id: str,
+        region_name: str,
+        cache_subnet_group_name: str,
+        cache_subnet_group_description: str,
+        subnet_ids: List[str],
+        tags: Optional[List[Dict[str, str]]],
+    ):
+        if tags is None:
+            tags = []
+        self.cache_subnet_group_name = cache_subnet_group_name
+        self.cache_subnet_group_description = cache_subnet_group_description
+        self.subnet_ids = subnet_ids
+        self.tags = tags
+
+        # Get VPC details from provided subnet IDs
+        # References rds models.py
+        ec2_backend = ec2_backends[account_id][region_name]
```

Comment:
Can we move the EC2 import down here? It's quite a computationally expensive import, so we should only do it when absolutely necessary.

(`from moto.ec2.models import ec2_backends`)

Reply:
sure! Good to know for other subnet-related features. That makes sense.